src/components/Footer.tsx

```diff
@@ -127,6 +127,12 @@ const Footer = ({
           >
             Français
           </LanguageSwitcherButton>
+          <LanguageSwitcherButton
+            variant={ButtonVariants.UNSTYLED}
+            onClick={handleChangeLanguage('de')}
+          >
+            Deutch
```

Comment:
Could we store these in a constant or enum?

Reply:
DONE